runtime/doc/api.txt

```diff
@@ -537,6 +537,12 @@ nvim__put_attr({id}, {start_row}, {start_col}, {end_row}, {end_col})
                 should probably be derived from a reformed bufhl/virttext
                 interface with full support for multi-line ranges etc
 
+nvim__screenshot({path})                                  *nvim__screenshot()*
```

Comment:
not in this PR, but later it should be made explicit automagically that every `nvim__` is unstable.